src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,8 @@ public Order completeOrder(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("successful fetched data from DB. UserID={}. Products are:", userId);
+        products.forEach(logger::info);
```

Comment:
```suggestion
         logger.info("successful fetched data from DB. UserID={}. Products are: {}", userId, products);
```

мне кажется делать отдельную запись в логе под каждый продукт не стоит, это сильно увеличит размер лог файла и такой лог будет тяжело читать